account-service/backend/app.py

```diff
@@ -0,0 +1,45 @@
+from flask import Flask, request, jsonify, session
+from flask_cors import CORS
```

Comment:
I feel that moving the backend outside of the frontend would be better and provides for cleaner separation.

Using our tutorial skeleton as a context, the `demo-app` folder is our frontend and `demo-service` folder is our backend.

The rough skeleton for both the frontend and backend are already in their respectively folders, so we would just have to shift the code and logic from this review into the respectively folders/files.

For reference, from chatgpt, the rough directory structure of our repo should look like this:
![image](https://github.com/user-attachments/assets/d0b3dae8-7090-431b-ab04-ad282ee81584)



And as for within the backend folder, the structure (including login logic from your review) should roughly look like this:
![image](https://github.com/user-attachments/assets/1832c55d-c21c-46fa-ad72-02650c681dec)



Please do bear in mind that their `frontend` folder = our `demo-app` folder, and their `backend` folder = our `demo-service` folder.

Sorry for asking you to perform a huge restructure.

Reply:
yep got it! will restrucutre accordingly